test/studies/adventOfCode/2021/bradc/futures/day18-bug2.bad

```diff
@@ -1,5 +1,7 @@
-$CHPL_HOME/modules/internal/OwnedObject.chpl:167: In initializer:
-$CHPL_HOME/modules/internal/OwnedObject.chpl:172: error: Could not coerce 'nil' to 'borrowed Node?' in initialization
-  $CHPL_HOME/modules/internal/OwnedObject.chpl:247: called as owned.init(type chpl_t = borrowed Node?)
+day18-bug2.chpl:26: warning: please use '?' when declaring a variable with generic type
+day18-bug2.chpl:26: note: for example with 'Node(?)'
+$CHPL_HOME/modules/internal/OwnedObject.chpl:50: In initializer:
+$CHPL_HOME/modules/internal/OwnedObject.chpl:55: error: Could not coerce 'nil' to 'borrowed Node?' in initialization
```

Comment:
This is a pretty significant line difference for what looks like the same call.  Was there a change to OwnedObject.chpl that didn't get committed?

Reply:
Nope. Line numbers are prediff'd away in .bad files so it just represents me updating it with current line numbers from other changes. But the line numbers will be ignored.